decidim-proposals/app/cells/decidim/proposals/participatory_text_proposal/show.erb

```diff
@@ -0,0 +1,9 @@
+<div class="row section hover-section">
```

Comment:
Hi there, @tramuntanal to get rid of the empty space between paragraphs, just remove `.section` class here

```suggestion
<div class="row hover-section">
```

Reply:
I've just applied your suggestion.